docs/space-daemon/getting-started.md

```diff
@@ -577,6 +577,47 @@ Returns a list of notifications objects. Notifications objects represent just sh
 
 ### Backup
 
+#### Get Usage Info
+
+> .getUsageInfo()
+
+> `[WIP]` <em>.getUsageInfo()</em>
+>
+> this method is still not supported by [space-daemon](https://github.com/FleekHQ/space-daemon)
```

Comment:
the method is already on [space-daemon master branch](https://github.com/FleekHQ/space-daemon/blob/master/grpc/proto/space.proto#L272), perhaps this is not needed it anymore?

Reply:
I removed it